Dockerfile

```diff
@@ -15,7 +15,6 @@ ENV BUILD_NUMBER ${BUILD_NUMBER:-1_0_0}
 
 RUN apt-get update && \
     apt-get -y upgrade && \
-    apt-get install -y curl && \
```

Comment:
we took this out of most projects a long while ago as it added extra dependencies that are sometimes vulnerable.